### ISSUE TITLE
Problem in error if data is not str

### ISSUE THREAD

**Issue by nikitinsm** [bcb/jsonrpcclient#56]
Response example:

```
{
  "jsonrpc": "2.0", 
  "id": 2, 
  "error": {
    "message": "some message", 
    "code": -32603, 
    "data": {
    }
  }
}
```

```Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/jsonrpcclient/client.py", line 213, in attr_handler
    return self.request(name, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/jsonrpcclient/client.py", line 200, in request
    return self.send(Request(method_name, *args, **kwargs))
  File "/usr/local/lib/python3.6/site-packages/jsonrpcclient/client.py", line 171, in send
    return self._send_message(request, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/jsonrpcclient/http_client.py", line 82, in _send_message
    log_format='<-- %(message)s (%(http_code)s %(http_reason)s)')
  File "/usr/local/lib/python3.6/site-packages/jsonrpcclient/client.py", line 114, in _process_response
    response['error'].get('data'))
  File "/usr/local/lib/python3.6/site-packages/jsonrpcclient/exceptions.py", line 64, in __init__
    [item for item in [message, data] if item is not None]))
TypeError: sequence item 1: expected str instance, dict found
```

**Comment by bcb** [comment]
Thanks for reporting. I've reproduced this error.

**Comment by bcb** [comment]
Pushed a fix to pypi in version 2.5.1.
